serdif/.htaccess

```diff
@@ -0,0 +1,2 @@
+RewriteEngine on
+RewriteRule (.*) https://github.com/navarral/serdif/$1 [R=301,L]
```

Comment:
Note this is going to the GitHub UI for those resources.  I see there's nothing there now.  You may consider some other hosting, like github pages, if you want to see the raw resources.

Reply:
Thanks for the suggestion. I will consider using GitHub pages. I generated the permanent URL link in advance since I have to include it in a journal paper by the end of the week. I will be adding the content during the next days. Thanks for your guidance.